packages/openapi-generator/src/generator.ts

```diff
@@ -74,94 +93,55 @@ export async function generate(options: GeneratorOptions): Promise<void> {
       ? 'Some clients could not be generated.'
       : 'Could not generate client.';
   await finishAll(promises, errorMessage);
+
+  if (options.perServiceConfig) {
+    writeFile(
+      options.perServiceConfig,
```

Comment:
This can be the path to the file or a folder. In case it is a folder a name should be set right?

Reply:
Ah I found that in the cli parse step this is already done.

Reply:
Correct 😊